project-5/main.py

```diff
@@ -137,19 +150,29 @@ def validation(self):
             # Close the cursor object
             cursor.close()
 
-            logging.info("created table successfully")
+            logging.info("data validated successfully")
         except Exception as e:
             raise ValidationError("validation failed") from e
 
-        self.insert()
+    def data_type_validation(self):
+        """
+        This function validates the data type
+        """
+        for num in range(len(self.df)):
+            if (self.df['Company Name'][num] == str or
+                    self.df['Company Link'][num] == str or
+                    self.df['Company LinkedIn'][num] == str):
```

Comment:
@Kong-Codes can you use the `type()` function to validate the data type like: `type(self.df['Company Name'][num]) == str

Reply:
Done i used isinstance() instead